cardano-db-tool/src/Cardano/Db/Tool/Validate/Balance.hs

```diff
@@ -63,25 +63,30 @@ getByronBalance addrText utxo = do
         then Just lovelace
         else Nothing
 
-getShelleyBalance
-    :: forall era. ShelleyBased era
-    => Text -> Shelley.UTxO era -> Either Text Word64
+getShelleyBalance :: forall era.
+                     ( Core.TxOut era ~ Shelley.TxOut era -- somewhere in ledger-spec, there is probably a better constraint synonym for these
```

Comment:
Can you try to match the formatting?

I wish I had a tool that could enforce formatting in a way that is not horrible, I hate brittany and ormul??? or whatever its called.